src/micromamba/main.cpp

```diff
@@ -159,6 +160,9 @@ init_network_parser(CLI::App* subcom)
 {
     subcom->add_option(
         "--ssl_verify", network_options.ssl_verify, "Enable or disable SSL verification");
+    subcom->add_option("--ssl_no_revoke",
```

Comment:
I would prefer using `--ssl-no-revoke`, and could you add the default at teh end of the description? (i.e. `(default: false)`